src/Microsoft.ML.Data/DataLoadSave/TransformWrapper.cs

```diff
@@ -45,39 +43,12 @@ public DataViewSchema GetOutputSchema(DataViewSchema inputSchema)
             _host.CheckValue(inputSchema, nameof(inputSchema));
 
             var dv = new EmptyDataView(_host, inputSchema);
-            var output = ApplyTransformUtils.ApplyAllTransformsToData(_host, _xf, dv);
+            var output = ApplyTransformUtils.ApplyTransformToData(_host, (IDataTransform)_xf, dv);
```

Comment:
>_xf [](start = 89, length = 3)

I would either change `_xf` to be an `IDataTransform`, or check in the ctor that `_xf is IDataTransform`.